components/eamxx/docs/index.md

```diff
@@ -1,10 +1,11 @@
-# The C++ E3SM Atmosphere Model (EAMxx)
+# The E3SM Atmosphere Model in C++ (EAMxx)
```

Comment:
How about "the E3SM Atmosphere Model ++ (EAMxx)" so it sounds like an incremental improvement on EAM rather than just a rewrite?

Reply:
Yeah, that's a good point, but it's official name is "EAM in C++". Hindsight 20/20 but I think it is too late for yet another rebranding.

Reply:
Yeah, that's a good point, but it's official name is "EAM in C++". Hindsight 20/20 but I think it is too late for yet another rebranding.

Reply:
👍